include/roco2/kernels/high_low.hpp

```diff
@@ -32,25 +47,41 @@ namespace kernels
             SCOREP_USER_REGION("high_low_bs_kernel", SCOREP_USER_REGION_TYPE_FUNCTION)
 #endif
             roco2::chrono::time_point deadline = std::chrono::high_resolution_clock::now();
+            // align across all procs
+            deadline -= deadline.time_since_epoch() % get_period();
+
+            // record highlow frequency to trace
+            roco2::metrics::frequency::instance().write(std::chrono::seconds(1) / std::chrono::duration_cast<std::chrono::duration<double>>(get_period()));
```

Comment:
The frequency metric is used to write the actual core/processor frequency. I think we could use the `utilization` metric instead.

Reply:
implemented in 3e199f6